### ISSUE TITLE
Python utility functions to get paths to thickness images by patient ID, study uid, code version

### ISSUE THREAD

**Issue by PaulHax** [PaulHax/knee-sarg#37]
Low priority as everthing is possible with `cartilage_thickness_runs.parquet` file

depends on #36 

For OAI, more convent to have args of `get_images(patient_id, study_month, left | right knee, code version=most_recent)`

prior art 
https://github.com/uncbiag/OAI_analysis/blob/master/collect_thickness_results.py
